### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by SiahRBLX** [xodamcm/BetterMobsOverworld#1]
Certain animals don't attack back like creeper's they punch instead of blowing up

**Comment by xodamcm** [comment]
yes I know, I wrote the mobs spawn, the experience, the different life, and the loots, ajouté quelques mobs, regler le knock back et quelques petites choses, easy things, for the rest I just took it from the original plugin: Mobs, I didn't write the mobs, I need contributor developer for this project
